include/panda_torque_mpc/reduce_collision_model.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <Eigen/Core>
+#include <boost/smart_ptr/shared_ptr.hpp>
+#include <chrono>
+#include <geometry_msgs/Pose.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <ratio>
+#include <ros/node_handle.h>
+#include <ros/package.h>
+#include <std_msgs/Header.h>
+#include <vector>
+
+#include <pinocchio/algorithm/model.hpp>
+#include <pinocchio/parsers/srdf.hpp>
+#include <pinocchio/parsers/urdf.hpp>
+#include <pinocchio/spatial/explog.hpp>
+#include <pinocchio/spatial/motion.hpp>
+#include <pinocchio/spatial/se3.hpp>
+#include <pinocchio/algorithm/geometry.hpp>
+#include <pinocchio/algorithm/joint-configuration.hpp>
+#include <pinocchio/fwd.hpp>
+#include <pinocchio/multibody/model.hpp>
+
+#include <hpp/fcl/collision_object.h>
+#include <hpp/fcl/shape/geometric_shapes.h>
+
+namespace panda_torque_mpc {
+boost::shared_ptr<pinocchio::GeometryModel> reduce_capsules_robot(
+    const boost::shared_ptr<pinocchio::GeometryModel> &collision_model) {
+  auto reduced_collision_model =
+      boost::make_shared<pinocchio::GeometryModel>(*collision_model);
+  std::vector<std::string> list_names_capsules;
+
+  for (pinocchio::GeomIndex geom_id = 0;
+       geom_id < (pinocchio::GeomIndex)collision_model->ngeoms; ++geom_id) {
+
+    pinocchio::GeometryObject &geometry_object =
+        collision_model->geometryObjects[geom_id];
```

Comment:
```suggestion
   auto &geometry_object = collision_model->geometryObjects[geom_id];
```
Can be a single line this way